tst/EnergyPlus/unit/SingleDuct.unit.cc

```diff
@@ -3003,8 +3000,8 @@ TEST_F(EnergyPlusFixture, VAVConstantVolTU_Reheat_Sizing)
     bool FirstHVACIteration = true;
     state->dataSingleDuct->sd_airterminal(SysNum).SimConstVol(*state,
                                                               FirstHVACIteration,
-                                                              state->dataSingleDuct->sd_airterminal(SysNum).ActualZoneNum,
-                                                              state->dataSingleDuct->sd_airterminal(SysNum).CtrlZoneNum);
+                                                              state->dataSingleDuct->sd_airterminal(SysNum).CtrlZoneNum,
+                                                              state->dataSingleDuct->sd_airterminal(SysNum).CtrlZoneInNodeIndex);
```

Comment:
Odd name choice of CtrlZoneInNodeIndex

void SingleDuctAirTerminal::SimConstVol(EnergyPlusData &state, bool const FirstHVACIteration, int const ZoneNum, int const ZoneNodeNum)